ui/src/components/executions/Overview.vue

```diff
@@ -6,8 +6,8 @@
             </el-col>
             <el-col :span="12" class="d-flex gap-2 justify-content-end">
                 <set-labels :execution="execution" />
-                <restart is-replay :execution="execution" class="ms-0" @follow="forwardEvent('follow', $event)" />
-                <restart :execution="execution" class="ms-0" @follow="forwardEvent('follow', $event)" />
+                <restart is-replay :initial-inputs="flow" :execution="execution" class="ms-0" @follow="forwardEvent('follow', $event)" />
```

Comment:
```suggestion
                <restart is-replay :flow="flow" :execution="execution" class="ms-0" @follow="forwardEvent('follow', $event)" />
```

If you're passing a flow, just call it flow, same for replay component